samples/buildstrategy/ko/buildstrategy_ko_cr.yaml

```diff
@@ -0,0 +1,83 @@
+---
+apiVersion: build.dev/v1alpha1
+kind: ClusterBuildStrategy
+metadata:
+  name: ko
+spec:
+  buildSteps:
+    - name: prepare
+      image: golang:1.15
+      securityContext:
+        runAsUser: 0
+        capabilities:
+          add: 
+            - CHOWN
+      command:
+        - /bin/bash
+      args:
+        - -c
+        - |
+          set -euo pipefail
+
+          chown -R 1000:1000 /workspace/output
+          chown -R 1000:1000 /workspace/source
+          chown -R 1000:1000 /tekton/home
+      resources:
+        limits:
+          cpu: 100m
+          memory: 128Mi
+        requests:
+          cpu: 100m
+          memory: 128Mi
+    - name: build-and-push
+      image: golang:1.15
+      workingDir: /workspace/source
+      securityContext:
+        runAsUser: 1000
+        runAsGroup: 1000
+      env:
+        - name: DOCKER_CONFIG
+          value: /tekton/home/.docker
+      command:
+        - /bin/bash
+      args:
+        - -c
+        - |
+          set -euo pipefail
+
+          # Parse image URL to extract repository and tag
+          IMAGE=$(build.output.image)
+
+          REPO=
+          TAG=
```

Comment:
Do we need these two empty `REPO=` and `TAG=` ? or do we need to set default TAG as `latest`?

Reply:
Leaving it empty will lead to not tag being passed to `ko` which will probably also mean `latest`.